### ISSUE TITLE
ARM cross compile error during nfclib-nci build

### ISSUE THREAD

**Issue by SangsubLim** [NXPNFCLinux/linux_libnfc-nci#119]
I checked the my NFC module on my raspberry pi by referring to the following document.
https://www.nxp.com/docs/en/application-note/AN11697.pdf

For reference, my NFC module is a USB type.
So I added --enable-lpcusbsio when running ./configure on raspbian.
During compilation, an error related to libudev occurred, so I ran sudo apt-get install libudev and it completed successfully.
After that, nfcDemoApp was run to test the operation.

I'm trying to develop on a PC with my ubuntu 16.04 installed, not raspbian.
So I downloaded the ARM GCC 8.3.0 version for the cross compile in Raspbian which is the same as the GCC version I am using.
After setting up the compiler, I did the same thing as working with raspbian.
Similarly, several errors related to libudev occurred.

In summary, what I would like to ask is:
Has anyone succeeded in ARM Cross compile with --enable-lpcusbsio applied?
If successful, ask how you resolved libudev.
I would be grateful if anyone could help me with this.
